src/sentry/templates/sentry/integrations/github-integration-exists-on-another-org.html

```diff
@@ -0,0 +1,24 @@
+{% extends "sentry/bases/modal.html" %}
+{% load i18n %}
+{% load sentry_assets %}
+{% load sentry_helpers %}
+
+{% block title %}{% trans "Github Integration Setup Failed" %} | {{ block.super }}{% endblock %}
+{% block wrapperclass %}narrow auth{% endblock %}
+
+{% block main %}
+{% script %}
+<script>
+  if (window.opener) {
+    window.opener.postMessage({{ payload|to_json }}, document.origin);
+  }
+  window.close();
+</script>
+{% endscript %}
+  <div class="align-center">
+    <p>
+      {% trans "It seems that your Github account has been installed on another Sentry organization. Please uninstall and try again." %}
```

Comment:
Should we assume it's the caller's GitHub account? You could go for a more generic "This GitHub account has been installed on another Sentry organization"

Reply:
@AniketDas-Tekky I think it's safe to assume that it is the caller's GitHub account, and we can assume they redirected from Github's configure page.

Reply:
Instead of "GitHub Account" should it be "GitHub App"?

Reply:
@AniketDas-Tekky This is validation for when the Github Sentry App is already installed for their Github Account for another Sentry organization. It looks like GIthub uses "Account" as a generic term and covers individuals and organizations. https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts